src/features/settings/settingTab.tsx

```
@@ -0,0 +1,254 @@
+		isRefetching,
+		isError,
+		error,
+	} = useQuery({
+		queryKey: ["pinecone-indexes"],
+		queryFn: fetchPineconeIndexes,
+	});
+
+	const isFetching = isLoading || isRefetching;
+
+	useEffect(() => {
+		if (pineconeIndexes && pineconeIndexes.length > 0) {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: plugin.saveSettings

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)